### ISSUE TITLE
It seems that the training model failed.

### ISSUE THREAD

**Issue by haoyuanzhe** [Zhenye-Na/DA-RNN#11]
Hi, Zhenye, thank you for excellent code. When i ran the code, it seemed that i got the wrong prediction result. 
![image](https://user-images.githubusercontent.com/42513621/60074880-91c37c00-9756-11e9-8ba2-a0648e672b0c.png) Could you please give some advice on how to overcome it? Thanks a lot.

**Comment by sherpahu** [comment]
I met the same problem. Have you fixed it?

**Comment by sherpahu** [comment]
I think I get the solution. The initial learning rate is too small in the command "python3 main --lr 0.0001 --epochs 50", so that the learning process is too slow.  
I change it to "python main.py --lr 0.1 --epochs 50", and it seems better.

**Comment by haoyuanzhe** [comment]
Thank you for your reply. But the prediction result still seems wrong? as shown in orange line and green line. I have find another implementation of DA-RNN and it can run successfully (https://github.com/Maple728/DA-RNN). Hope this helps. :)

**Comment by sherpahu** [comment]
Thank you!
大家都是中国人，一直用英语感觉好尬

**Comment by haoyuanzhe** [comment]
哈哈哈，我昨儿就想用中文了。

**Comment by Zhenye-Na** [comment]
Problem solved. Please feel free to check it out.
